flask/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask==3.0.0
+mariadb==1.1.8
+python-dotenv==1.0.0
```

Comment:
is this package used anywhere? I thought we were loading environment variables in via docker

Reply:
oh yeah, i was using that for development, before we decided to put the env in the docker-compose.